docs/howto/platform-sdk-quickstart-guide.md

```diff
@@ -0,0 +1,194 @@
+
+1.  To start on the user interface of your project, navigate to the
+    folder that was created for your workspace in the last step of the
+    CLI section above. In this workspace folder, there is a UI directory
+    for the workspace. There is a folder inside of this directory with
+    the same name as the plugin you created.
+2.  Navigate to [\[workspace directory\]\\ui\\\[plugin
+    name\]\\src\\src]{.path} for the tool you are working on.
+3.  Open the index.tsx file. This is the primary
+    [React](https://reactjs.org/)
+    file for the tool UI.
+4.  Make a change to the file. For example, edit the text of a tag, or
+    add a [new
+    component](https://alteryx.github.io/alteryx-ui/)
+    or HTML element to the page.
+5.  Save the file.
+6.  For a quick preview of your tool in a browser, run this command:\
+    \
+    `npm run start`
+7.  To view your tool in Alteryx Designer, run this command from the
+    root workspace directory. This command builds and installs your
+    plugin into Designer.\
+    \
+    `ayx_plugin_cli designer-install`
+
+Open Alteryx Designer, search for your tool, place it on the canvas, and
+view the changes.
+
+For more information about the UI SDK, please visit the [reference
+documents](https://alteryx.github.io/alteryx-ui/).
+
+
+### Python SDK
+
+#### Warning
+V1 of AYX Python SDK is only compatible with Alteryx Designer versions
+2021.2 and 2021.3. For the best experience, and to ensure compatibility
+with the most recent versions of Designer, please use [version 2 of AYX
+Python
+SDK](https://help.alteryx.com/developer-help/ayx-python-sdk-v2).
+
+To learn more about version 2 of the AYX Python SDK, go to [Changes and
+Improvements to Python SDK in
+v2.0.0](https://help.alteryx.com/developer-help/changes-and-improvements-python-sdk-v2).
+
+
+The Python SDK defines the way that you should create the engine/back
+end/runtime of your tool. The back end also receives the configuration
+values that were created by the front end of the tool in the UI SDK
+section.
+
+Follow these steps to get started with the Python SDK:
+
+1.  To start on the engine or runtime of your project, navigate to the
+    folder that was created for your workspace in the last step of the
+    CLI section above. In this workspace folder, there is a back-end
+    directory for the workspace.
+2.  Navigate to [\[workspace directory\]\\backend\\ayx_plugins\\\[plugin
+    name\].py]{.path} for the tool you are editing.
+3.  Make changes to the file.
+4.  Save the file.
+5.  Run this command from the root workspace directory. This builds and
+    installs your plugin into Alteryx Designer.\
+    \
+    `ayx_plugin_cli designer-install`
+
+Open Alteryx Designer, search for your tool, place it on the canvas, and
+view the changes.
+
+For more information about the Python SDK, please go to the [reference
+documents](https://help.alteryx.com/developer-help/ayx-python-sdk) or access the full [Python
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version